save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/TableComponent.kt

```diff
@@ -97,6 +98,7 @@ fun <D : Any, P : TableProps<D>> tableComponent(
     initialPageSize: Int = 10,
     useServerPaging: Boolean = false,
     usePageSelection: Boolean = false,
+    isInvisibleTable: Boolean = false,
```

Comment:
Did you mean something like `isTransparentGrid`?

Reply:
Renamed